src/main/java/com/hcl/appscan/jenkins/plugin/scanners/DynamicAnalyzer.java

```diff
@@ -325,17 +326,22 @@ public FormValidation doCheckScanFile(@QueryParameter String scanFile) {
 			return FormValidation.ok();
 		}
 
-		public FormValidation doCheckTarget(@QueryParameter String target,@RelativePath("..") @QueryParameter String credentials, @AncestorInPath ItemGroup<?> context, @QueryParameter String presenceId) {
+		public FormValidation doCheckTarget(@QueryParameter String target,@RelativePath("..") @QueryParameter String credentials, @AncestorInPath ItemGroup<?> context, @QueryParameter String presenceId) throws IOException {
```

Comment:
None of the Formvalidations are throwing IO Exception why is it required ?

Reply:
Removed the IO exception.